src/store/actions/keymap.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from '@angular/core';
+import { Action } from '@ngrx/store';
+
+import { Keymap } from '../../config-serializer/config-items/Keymap';
+
+@Injectable()
+export class KeymapActions {
+    static PREFIX = '[Keymap] ';
```

Comment:
If I know right, then it is not allowed to have static const in class. I suggest to create a namespace with these consts in it, or are the strings used or only the symbol? If only the symbol, then you could create an enum from them.